apps/prairielearn/src/tests/groupRole.test.ts

```diff
@@ -1516,14 +1475,11 @@ describe('Test group role reassignments with role of minimum > 1', function () {
     });
 
     step('should have correct errors displayed', function () {
-      elemList = locals.$(
-        '.alert:contains(1 more student needs to be assigned to the role "Recorder" (Found 1, expected exactly 2).)',
-      );
-      assert.lengthOf(elemList, 0);
-      elemList = locals.$(
-        '.alert:contains(At least one user does not have a role. All users must have a role.)',
+      assertAlert(
+        locals.$,
+        '1 more student needs to be assigned to the role "Recorder" (1 assigned, 2 expected)',
```

Comment:
This seems to be a case where a recent refactor fixed this test the wrong way. The error should indeed display, but the text was incorrect.